SumBenchDockerfile

```diff
@@ -0,0 +1,14 @@
+FROM clipper/py-rpc:latest
+
+COPY containers/python/sum_container.py /containers/python/
+COPY bench/setup_sum_bench_docker.sh /bench/
+COPY bench/set_bench_env_vars.sh /bench/
+COPY containers/python/rpc.py /containers/python/
```

Comment:
Without copying rpc.py, I get an ImportError from the import statement in `sum_container.`py. Same thing happens for NoopBenchDockerfile with `noop_container.py`. Was having a hard time debugging it -- I'll keep looking into it and push a fix. Figured I'd this push up set of changes and not let this one problem block reviews.